src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const lis = tree.querySelectorAll('li');
+
+lis.forEach((el) => {
+  const text = el.childNodes[0].nodeValue.trim();
+  const span = document.createElement('span');
+
+  span.textContent = text;
+  el.childNodes[0].replaceWith(span);
+});
+
+tree.addEventListener('click', (ev) => {
+  ev.stopPropagation();
+
+  const span = ev.target.closest('span');
+
+  if (!span) {
+    return;
+  }
+
+  const ul = span.nextElementSibling;
```

Comment:
Ensure that `span.nextElementSibling` is indeed a `ul` element before attempting to toggle its display style. This will prevent errors if the structure is not as expected.